Resources/Computing/ARCComputingElement.py

```diff
@@ -423,6 +423,8 @@ def getJobOutput( self, jobID, localDir = None ):
 
     isItOkay = job.Retrieve( self.usercfg, arc.URL( workingDirectory ), False )
     if ( isItOkay ):
+      if not os.path.isfile(outFileName):
```

Comment:
WOW - this is something I would not have been able to write. Will copy-paste your solution and test it out.